jwql/tests/test_bad_pixel_monitor.py

```diff
@@ -23,20 +23,18 @@
 
 from jwst.datamodels import dqflags
 
-from jwql.database.database_interface import NIRCamBadPixelQueryHistory, NIRCamBadPixelStats
-from jwql.database.database_interface import NIRISSBadPixelQueryHistory, NIRISSBadPixelStats
-from jwql.database.database_interface import MIRIBadPixelQueryHistory, MIRIBadPixelStats
-from jwql.database.database_interface import NIRSpecBadPixelQueryHistory, NIRSpecBadPixelStats
-from jwql.database.database_interface import FGSBadPixelQueryHistory, FGSBadPixelStats
-
-
-ON_JENKINS = '/home/jenkins' in os.path.expanduser('~')
-
-if not ON_JENKINS:
+ON_GITHUB_ACTIONS = '/home/runner' in os.path.expanduser('~')
+
+if not ON_GITHUB_ACTIONS:
+    from jwql.database.database_interface import NIRCamBadPixelQueryHistory, NIRCamBadPixelStats
+    from jwql.database.database_interface import NIRISSBadPixelQueryHistory, NIRISSBadPixelStats
+    from jwql.database.database_interface import MIRIBadPixelQueryHistory, MIRIBadPixelStats
+    from jwql.database.database_interface import NIRSpecBadPixelQueryHistory, NIRSpecBadPixelStats
+    from jwql.database.database_interface import FGSBadPixelQueryHistory, FGSBadPixelStats
     from jwql.instrument_monitors.common_monitors import bad_pixel_monitor
 
 
-@pytest.mark.skipif(ON_JENKINS, reason='Jenkins server currently unable to import jwst_reffiles')
+@pytest.mark.skipif(ON_GITHUB_ACTIONS, reason='Github Actions server currently unable to import jwst_reffiles')
```

Comment:
Things have changed since we first wrote this, and `jwst_reffiles` is now `pip` installable and should be part of the nominal 3.7 and 3.8 environments, so perhaps these can be removed.  It would be a good thing to test out.

Reply:
Hey @bourque  sounds good. Do you mean, remove the tests that mimic JWST ref files and just import `jwst_reffiles`?

Reply:
Hey @bourque, I noticed that even though you can use `jwst_reffiles` in 3.7 and 3.8, these tests still call monitors which we can't currently import. So, I just went ahead and changed the reason message.